src/app/fhir-validator/service/fhir-validator.service.ts

```diff
@@ -2,15 +2,15 @@ import { Injectable } from '@angular/core';
 import {BehaviorSubject, Observable, Subject} from "rxjs";
 import {HttpClient, HttpHeaders} from "@angular/common/http";
 import {map} from "rxjs/operators";
-import {ValidationResults} from "../domain/ValidationResoults";
+import {ValidationResults} from "../domain/ValidationResults";
+import {ValidatorConstants} from "../providers/validator-constants";
 
 @Injectable({
   providedIn: 'root'
 })
 export class FhirValidatorService {
 
-  private prodUri = "https://gt-apps.hdap.gatech.edu/HL7ValidatorService/fhir";
-  //private prodUri = "http://127.0.0.1:8080/fhir/$validate";
+  private prodUri = ValidatorConstants.prodUri;
```

Comment:
See the comments above, the ValidatorConstants.prodUri should be ValidatorConstants.PROD_URI